docs/spec.md

```diff
@@ -495,6 +498,26 @@ Unsubscribe from [connection graph updates](#connection-graph-update). Only supp
 }
 ```
 
+### Fetch asset
+
+Fetch an asset from the server. Only supported if the server previously declared that it has the `assets` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"fetchAsset"`
+- `uri`: string, unique identifier to locate the asset
```

Comment:
No, it does not have to be an identifier following the URI spec, it can be any string that identifies an asset uniquely. Not sure what a better field name would be, maybe just `id` ?

Reply:
What would the client do with that info? At the end of the day the server is going to be the one who decides whether the request is valid or not.

Reply:
The client could use this info to determine what a valid url would be. I was thinking e.g. about [this](https://github.com/foxglove/studio/blob/28ab637201c23163980f0bdd2145a27ddc532e3a/packages/studio-base/src/panels/ThreeDeeRender/renderables/Urdfs.ts#L823-L832)